app/src/main/java/org/oppia/android/app/fragment/BUILD.bazel

```diff
@@ -91,3 +91,18 @@ kt_android_library(
         "//third_party:androidx_appcompat_appcompat",
     ],
 )
+
+kt_android_library(
```

Comment:
please include this target in [app module library in app/BUILD.bazel](https://github.com/yash10019coder/oppia-android/blob/d8ab9e953f694e21fb6621ef7786516f9f068bce/app/BUILD.bazel#L734)  as the dependency. It's required fragment for the app.

Reply:
Added, Thanks